app/controllers/account/admin/billing_contracts_controller.rb

```diff
@@ -5,6 +5,15 @@ class Account::Admin::BillingContractsController < Account::Admin::AdminControll
 
   def index
     @billing_contracts = @company.billing_contracts.page params[:page]
+    respond_to do |format|
```

Comment:
Enter

Reply:
rubocop does not agree with you)

Reply:
![Screenshot from 2020-11-07 17-08-51](https://user-images.githubusercontent.com/31774981/98444747-116ba400-211c-11eb-8568-8c734d8b5d5c.png)